_example/update_user.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"log"
+	"os"
+
+	"github.com/zoom-lib-golang/zoom-lib-golang"
+)
+
+func main() {
+	var (
+		apiKey    = os.Getenv("ZOOM_API_KEY")
+		apiSecret = os.Getenv("ZOOM_API_SECRET")
+		userID    = os.Getenv("USER_ID")
+	)
+
+	zoom.APIKey = apiKey
+	zoom.APISecret = apiSecret
+
+	// Get User info by UserID
+	getOpt := zoom.GetUserOpts{
+		EmailOrID: userID,
+	}
+
+	response, err := zoom.GetUser(getOpt)
```

Comment:
Would like to see you use `user` instead of `response` here as I think that would be more clear. When I saw you setting `response.FirstName` below, I was a little confused until I looked more carefully.